### ISSUE TITLE
Unable to DL or ignore video in DL queue 

### ISSUE THREAD

**Issue by PhuriousGeorge** [tubearchivist/tubearchivist#237]
I can't seem to dispose of video MblM_-JZPJs from my DL queue wether by attempting to DL or ignore.  Is there a way to manually kill this entry?

```
ignore video MblM_-JZPJs
{"error":{"root_cause":[{"type":"document_missing_exception","reason":"[_doc][MblM_-JZPJs]: document missing","index_uuid":"VuwCFmjLQlGWbqzVsznj5g","shard":"0","index":"ta_download"}],"type":"document_missing_exception","reason":"[_doc][MblM_-JZPJs]: document missing","index_uuid":"VuwCFmjLQlGWbqzVsznj5g","shard":"0","index":"ta_download"},"status":404}
```

**Comment by bbilly1** [comment]
That fix is incoming. In the meantime, if you want to manually delete an entry from the download queue you can run this from within the container:

```
curl -XDELETE "$ES_URL/ta_download/_doc/MblM_-JZPJs?pretty" -u elastic:$ELASTIC_PASSWORD
```

This will delete *MblM_-JZPJs* if it's there.

**Comment by PhuriousGeorge** [comment]
Seems the command located the entry, but didn't remove it:
```
root@0c2fb5011ad0:/app# curl -XDELETE "$ES_URL/ta_download/_doc/MblM_-JZPJs?pretty" -u elastic:$ELASTIC_PASSWORD
{
  "_index" : "ta_download",
  "_type" : "_doc",
  "_id" : "MblM_-JZPJs",
  "_version" : 3,
  "result" : "not_found",
  "_shards" : {
    "total" : 1,
    "successful" : 1,
    "failed" : 0
  },
  "_seq_no" : 127697,
  "_primary_term" : 13
}
```
Sorry, not so familiar with ES yet.

**Comment by PhuriousGeorge** [comment]
For reference from our Discord convo:

Somehow the document ID is a premium video but the YTID that was originally grabbed was not and is now unlisted
```
      {
        "_index" : "ta_download",
        "_type" : "_doc",
        "_id" : "uHuk_0lCkPo",
        "_score" : 40.841564,
        "_source" : {
          "youtube_id" : "MblM_-JZPJs",
          "channel_name" : "Peter Hollens",
          "vid_thumb_url" : "https://i.ytimg.com/vi_webp/MblM_-JZPJs/maxresdefault.webp",
          "title" : "Trailer Peter Hollens YouTube Premium afterparty",
          "channel_id" : "UCgITW_70LNZFkNna7VsXbuQ",
          "channel_indexed" : true,
          "duration" : "00:30",
          "published" : "2022-03-15",
          "timestamp" : 1650611219,
          "status" : "pending"
        }
      }
```

**Comment by bbilly1** [comment]
That is now fixed in v0.1.5. Thanks for helping debug this.
